test-fixtures/update_application_services_version.py

```diff
@@ -0,0 +1,79 @@
+import os, json
+import requests
+import datetime
+from pprint import pprint
+from enum import Enum
+
+
+APPLICATION_SERVICES_TAGS_INFO = 'https://api.github.com/repos/mozilla/application-services/tags'
+CARTFILE = 'Cartfile'
+
+def available_tags():
+    try:
+        resp = requests.get(APPLICATION_SERVICES_TAGS_INFO, headers={'Authorization': 'Token {}'.format(secret.GITHUB_TOKEN)})
+        resp_json = resp.json()
+        print (resp_json[0])
+        return resp_json[0]
+    except HTTPError as http_error:
+        print('An HTTP error has occurred: {http_error}')
+    except Exception as err:
+        print('An exception has occurred: {err}')
+
+def latest_version():
+    tags = available_tags()
+    latest_tag_version = tags['name']
+    return latest_tag_version
+
+def read_cartfile_tag_version():
+    # Read Cartfile to find the current a-s version
+    with open(CARTFILE) as fp:
```

Comment:
We may need to add the ../ somehow since it is in a parent directory but it was not needed for similar script for bitrise file.. so not sure here either..

Reply:
Can debug with ```os.getcwd()```

Reply:
According to my tests locally, the path is the root path, we don't need to modify the path to Cartfile...